src/internalHelpers/_parseColorString.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+/** */
+type RgbColor = {
+  red: number,
+  green: number,
+  blue: number,
+  alpha?: number,
+}
+
+const hexRegex = new RegExp('^#[a-fA-F0-9]{6}$')
+const reducedHexRegex = new RegExp('^#[a-fA-F0-9]{3}$')
+// eslint-disable-next-line no-useless-escape
```

Comment:
Why do we need to disable that rule? There's something in the regex below that shouldn't be escaped, let's not escape it?